x-pack/plugins/maps/public/components/layer_panel/filter_editor/_filter_editor.scss

```diff
@@ -0,0 +1,4 @@
+.mapFilterEditorModalBody {
+  width: 85vw;
```

Comment:
Watch out for this on mobile you might be overriding the full screen modal at small screens.